src/paginate.ts

```diff
@@ -119,6 +121,11 @@ export function getFilterTokens(raw: string): string[] {
 
 function parseFilter<T>(query: PaginateQuery, config: PaginateConfig<T>) {
     const filter: { [columnName: string]: FindOperator<string> } = {}
+    let filterableColumns = config.filterableColumns
```

Comment:
just noticed now, variable is unused. please remove if not needed

Reply:
Oh sorry, I forgot to replace a later occurence of `config.filterableColumns` with just `filterableColumns`